spec/system/placements/placements/searching_for_a_placement_spec.rb

```diff
@@ -60,7 +61,7 @@
       and_i_fill_in_location_search_with("Guildford")
       and_i_click_on("Apply filters")
       then_i_see_placements_for(school_name: "Guildford School", list_item: 0, distance: 0.0)
-      and_i_see_placements_for(school_name: "London School", list_item: 1, distance: 26.7)
+      then_i_see_placements_for(school_name: "London School", list_item: 1, distance: 26.7)
```

Comment:
I think you're missing tests to check the drive, walking and transit times appear.